src/main/java/org/folio/domain/strategy/ItemStatusBasedTenantPickingStrategy.java

```diff
@@ -38,22 +33,31 @@ public class ItemStatusBasedTenantPickingStrategy implements TenantPickingStrate
   private final SearchClient searchClient;
 
   @Override
-  public Optional<String> pickTenant(String instanceId) {
+  public List<String> pickTenants(String instanceId) {
```

Comment:
We are not picking one tenant anymore, so "get" or "find" is more appropriate
```suggestion
  public List<String> findTenants(String instanceId) {
```